src/PrestaShopBundle/Controller/Admin/Improve/International/CurrencyController.php

```diff
@@ -153,6 +153,7 @@ public function createAction(Request $request)
         return $this->render('@PrestaShop/Admin/Improve/International/Currency/create.html.twig', [
             'isShopFeatureEnabled' => $multiStoreFeature->isUsed(),
             'currencyForm' => $currencyForm->createView(),
+            'currencyName' => null !== $currencyForm ? array_values($currencyForm->getData()['names'])[0] : null,
```

Comment:
I'm not sure it is really useful for the creation form You can't know in advance which currency will be created so in the data will be empty Maybe we should have a text like `Create new currency` or something similar
@LouiseBonnard what do you think?

Reply:
I suggest either _Currency_ or _Name_. _Name_ is the most consistent option since most of the creation forms in the software start with this wording.

Reply:
I dont really get your point @LouiseBonnard

Reply:
Decision have been made to keep *Currencies* for creation and update forms

Done